pathfinding_service.rst

```diff
@@ -231,6 +231,10 @@ If no possible path is found, one of the following errors is returned:
 * No suitable path found
 * Rate limit exceeded
 * From or to invalid
+* The token network address is invalid
+* Bias is invalid
+* Max_path is invalid
```

Comment:
s/Max_path/max_paths

Reply:
I wanted to stay in line with the previous errors. If we change that, then it should be
```
* No suitable path found
* Rate limit exceeded
* "from" or "to" invalid
* The "token_network_address" is invalid
* "bias" is invalid
* "max_path" is invalid
```